mpsl/include/mpsl_fem_protocol_api.h

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file mpsl_fem_protocol_api.h

```

Comment:
Do I have to use general MPSL API to use its FEM part? Like ``mpsl_init``?

Is it possible to make FEM part of MPSL standalone library in the nrfxlib or in any other repository that is part of NCS? I imagine that some users would like to use Front End Module Control API without actually using MPSL because they develop a single protocol application.